### PR TITLE
Move python package to python directory

### PR DESCRIPTION
**Pull request by andrewdotn**
This is to make room for a new node package, that’ll be built from the same `hfst-optimized-lookup.cc` file, but will have different glue code.

### CODE REVIEW

**Comment by andrewdotn** [comment]
The PR to add the node package, which could be created once this is merged, looks like this:

https://github.com/andrewdotn/hfst-optimized-lookup/compare/python-dir...andrewdotn:node?expand=1

File: hfst-optimized-lookup.cc [hunk 1]
```diff
@@ -253,6 +253,7 @@ int main(int argc, char **argv)
 
         case 'e':
           echoInputsFlag = true;
+          break;
```
Comment (eddieantonio) [inline hfst-optimized-lookup.cc:256]: Is this bug in upstream? Not that you need to give them a PR, I'm just curious.

**Review by eddieantonio — APPROVED** [review]
LGTM!

File: hfst-optimized-lookup.cc (hunk 1 above)
Comment (andrewdotn) [reply hfst-optimized-lookup.cc:256]: Good call: https://github.com/hfst/hfst/pull/534

**Pull request merged**